tools/eslint-rules/rules-utils.js

```diff
@@ -24,6 +24,16 @@ module.exports.isBinding = function(node, modules) {
   }
 };
 
+/**
+ * Return true if common module is required
+ * in AST Node under inspection
+ */
+var commonModuleRegExp = new RegExp(/^(\.\.\/)*common(\.js)?$/);
+module.exports.isCommonModule = function(node) {
+  return node.callee.name === 'require' &&
+         commonModuleRegExp.test(node.arguments[0].value);
```

Comment:
Here the access to the first argument has to be guarded. In case a node has no arguments, this will trigger an error.

So please add a `node.arguments.length !== 0 &&` in the middle.